packages/fxa-auth-server/config/dev.json

```diff
@@ -359,7 +359,7 @@
         "termsUri": "",
         "privacyUri": "",
         "trusted": true,
-        "allowedScopes": "https://identity.mozilla.com/account/subscriptions",
+        "allowedScopes": "https://identity.mozilla.com/account/subscriptions https://identity.mozilla.com/account/newsletters",
```

Comment:
> 
> 
> Where do these `clients` objects live in non-dev contexts?

They are in the database.

Reply:
> 
> 
> Seems like it might be in fxa-content-server/server/lib/configuration.js? I don't understand why they would live on two different servers though.

The ones in the content-server are used when creating an access token for Payments before it redirects users there.